assets/csidriveroperators/vsphere/06_clusterrole.yaml

```diff
@@ -276,18 +276,13 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - get
+  - '*'
 - apiGroups:
   - cns.vmware.com
   resources:
-  - cnsvspherevolumemigrations
+  - '*'
   verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-  - delete
+  - '*'
```

Comment:
This is for enabling webhook right?

Reply:
Correct, to make sure the webhook can start we need to enable the CSI migration in the driver; and once we do that the driver starts meddling with objects from this api group.